src/wp-includes/general-template.php

```diff
@@ -491,7 +492,7 @@ function wp_registration_url( $redirect = '' ) {
 	 *
 	 * @param string $register The user registration URL.
 	 */
-	return apply_filters( 'register_url', $register_url, $redirect );
+	return apply_filters( 'register_url', $register_url );
```

Comment:
Adding `$redirect_to` as additional parameter makes actual sense to me, but maybe as a follow up ticket to streamline discussion?

Reply:
It may indeed be subject to further discussion, so we can hold off on adding `$redirect_to` for now and consider it for a follow-up ticket.